deliveries/KZero_m2.md

```diff
@@ -0,0 +1,29 @@
+# Milestone Delivery :mailbox:
+
+> ⚡ Only the GitHub account that submitted the application is allowed to submit milestones. 
+> 
+> Don't remove any of the mandatory parts presented in bold letters or as headlines! Lines starting with `>`, such as this one, can be removed.
+
+**The delivery is according to the official [milestone delivery guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/milestone-deliverables-guidelines.md).**  
+
+* **Application Document:** [KZero Grant Application](https://github.com/w3f/Grants-Program/blob/master/applications/KZero.md)
+* **Milestone Number:** 2
+
+## KZero zk circuit implementation
+
+**Deliverables**
+Number	Deliverable	Specification
+| Number | Deliverable               | Specification                                                |
+| ------ | ------------------------- | ------------------------------------------------------------ |
+| 0a.    | License                   | GPLv3                                                        |
+| 0b.    | Documentation             | We will provide both inline documentation of the code and a basic tutorial that explains how a user can (for example) spin up one of our Substrate nodes and send test transactions, which will show how the new functionality works. |
+| 0c.    | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| 0d.    | Docker                    | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. |
```

Comment:
@dejavukong Could you link to the docker file(s) & setup instructions here?

Reply:
@takahser Good question. During the M2 delivery, we modified the architecture once to pursue precise calculation of transaction fees, which made it no longer easy to construct zklogin transactions using only the polkadotjs app. Therefore, we would like to move this dockerfile deliverable to M3, where it will be easier to test the functionality in combination with the web interface and SDK in M3.

Reply:
@takahser I've reviewed the proposal application, and there was no commitment to deliver a dockerfile in M2.